tests/verkle/eip4762_verkle_gas_witness/test_contract_execution.py

```diff
@@ -228,16 +231,17 @@ def test_contract_execution(
     )
 
     code_chunks = chunkify_code(bytecode)
-    assert len(code_chunks) > 1
+    assert len(code_chunks) > 0
 
     witness_check = WitnessCheck()
     for address in [TestAddress, TestAddress2, env.fee_recipient]:
         witness_check.add_account_full(
             address=address,
             account=(None if address == env.fee_recipient else pre[address]),
         )
-    for chunk_number in witness_code_chunk_numbers:
-        witness_check.add_code_chunk(TestAddress2, chunk_number, code_chunks[chunk_number])
+    for chunk_ranges in witness_code_chunk_numbers:
+        for chunk_number in range(chunk_ranges[0], chunk_ranges[1] + 1):
+            witness_check.add_code_chunk(TestAddress2, chunk_ranges, code_chunks[chunk_number])
```

Comment:
```suggestion
            witness_check.add_code_chunk(TestAddress2, chunk_number, code_chunks[chunk_number])
```
this should fix it I believe :)